scipy/stats/tests/test_marray.py

```diff
@@ -7,6 +7,7 @@
 from scipy.stats._axis_nan_policy import _axis_nan_policy_factory
 
 
+pytest.skip(allow_module_level=True)  # temporary until `marray` supports `device`
```

Comment:
`marray` 0.0.9 added `device`, but broke mdhaber/marray#123, so I pinned us at 0.0.8. I'll fix that soon (thoughts on the best way to do that?) and remove these skips.